scripts/nightly_bench.py

```
@@ -0,0 +1,535 @@
+
+    calibrate_timeout()
+    build()
+    meta = commit_info()
+    rows, skipped = run_sweep()
+
+    payload = {
+        "generated_at": datetime.now(timezone.utc).isoformat().replace("+00:00", "Z"),
+        **meta,
+        "min_bench_seconds": MIN_BENCH_SECONDS,
+        "run_timeout_seconds": RUN_TIMEOUT,
+        "configs": [{"key": k, "label": l, "threads": t, "proof": p} for k, l, t, p in CONFIGS],
```

Ruff (0.15.18)

[error] Ambiguous variable name: `l`

(E741)